Source/Shaders/PolylineShadowVolumeVS.glsl

```diff
@@ -0,0 +1,123 @@
+attribute vec3 position3DHigh;
+attribute vec3 position3DLow;
+attribute float batchId;
+
+varying vec4 v_startPlaneEC;
```

Comment:
You should probably limit the number of varyings to 8. [MAX_VARYING_VECTORS](https://webglstats.com/webgl/parameter/MAX_VARYING_VECTORS)

Reply:
Nevermind, the fragment shader only uses 7.

Reply:
I can trim it down... the polyline material shaders also swallow a couple varyings that don't show in the FS.